var/spack/repos/builtin/packages/arm-perflibstools/package.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class ArmPerflibstools(MakefilePackage):
+    """This project provides tools to enable users of HPC applications to understand
+    which routines from Arm Performance Libraries are being called."""
+
+    homepage = "https://github.com/ARM-software/perf-libs-tools"
+    git      = "https://github.com/ARM-software/perf-libs-tools.git"
+
+    # notify when the package is updated.
+    maintainers = ['OliverPerks', 'chrisgoodyer']
+
+    version('master', branch='master')
+
+    depends_on('python@3:')
+
+    # Patch to remove xblas.h include and non gcc routines
+    patch('simple.patch')
+
+    def edit(self, spec, prefix):
+        makefile = FileFilter('Makefile')
+        makefile.filter(r'\s*CC\s*=.*',  'CC = '  + spack_cc)
+
+    def install(self, spec, prefix):
+        make()
+
+        # Install
+        install_tree('lib', prefix.lib)
+        install_tree('tools', prefix.bin)
+
+    def setup_run_environment(self, env):
+        env.prepend_path("PYTHONPATH", join_path(self.prefix, "bin"))
```

Comment:
I think you can avoid this if you say that this package:
```python
extends('python')
```
In any case extending Python is the right thing to declare when a package provides a new Python module / package.

Reply:
Awesome, I didnt know about this, so thanks.